firmware/klipper_configurations/Octopus/Voron2_Octopus_Config.cfg

```diff
@@ -0,0 +1,646 @@
+uart_pin: PC6
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z1 Stepper - Rear Left
+##  Connected to MOTOR_3
+[stepper_z1]
+step_pin: PG4
+dir_pin: !PC1
+enable_pin: !PA0
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z1]
+uart_pin: PC7
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z2 Stepper - Rear Right
+##  Connected to MOTOR_4
+[stepper_z2]
+step_pin: PF9
+dir_pin: PF10
+enable_pin: !PG2
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z2]
+uart_pin: PF2
+interpolate: true
+run_current: 0.8
+hold_current: 0.80
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z3 Stepper - Front Right
+##  Connected to MOTOR_5
+[stepper_z3]
+step_pin: PC13
+dir_pin: !PF0
+enable_pin: !PF1
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z3]
+uart_pin: PE4
+interpolate: true
+run_current: 0.8
+hold_current: 0.80
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+
+#####################################################################
+# 	Extruder
+#####################################################################
+
+#	Connected to MOTOR_0
+#   Heater - HE0
+#   Thermistor - T0
+[extruder]
+step_pin: PE2
+dir_pin: PE3
+enable_pin: !PD4
+##	Update value below when you perform extruder calibration
+##	If you ask for 100mm of filament, but in reality it is 98mm:
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  22.6789511 is a good starting point
+rotation_distance: 22.6789511	#Bondtech 5mm Drive Gears
+##	Update Gear Ratio depending on your Extruder Type
+##	Use 50:17 for Afterburner/Clockwork (BMG Gear Ratio)
+##	Use 80:20 for M4, M3.1
+gear_ratio: 50:17				#BMG Gear Ratio
+microsteps: 16
+full_steps_per_rotation: 200	#200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: PA2
+##	Validate the following thermistor type to make sure it is correct
+sensor_type: ATC Semitec 104GT-2
+sensor_pin: PF4
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+min_extrude_temp: 170
+
+##	Try to keep pressure_advance below 1.0
+pressure_advance: 0.05
+##	Default is 0.040, leave stock
+pressure_advance_smooth_time: 0.040
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 extruder]
+uart_pin: PE1
+interpolate: false
+run_current: 0.5
+hold_current: 0.4
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+#####################################################################
+# 	Bed Heater
+#####################################################################
+
+[heater_bed]
+##	SSR Pin - HE1
+##  Thermistor - TB
+heater_pin: PD14
```

Comment:
I think this should be PA1 for the SSR

Reply:
HE1 is actually PA3, but good catch.

I don't see the point in using BED_OUT for the SSR. it's extra wiring for no gain.